libs/telemetry/src/main/java/org/opensearch/telemetry/tracing/DefaultTracer.java

```diff
@@ -27,31 +27,37 @@ class DefaultTracer implements Tracer {
     static final String THREAD_NAME = "th_name";
 
     private final TracingTelemetry tracingTelemetry;
-    private final TracerContextStorage<String, Span> tracerContextStorage;
+    private final TracerContextStorage<String, Span> spanTracerContextStorage;
```

Comment:
Could you please revert the rename? Difficult to understand what has changed

Reply:
Reverted the rename changes.